Modules/ThirdParty/PNG/src/itkpng/png.h

```diff
@@ -1974,6 +1988,46 @@ PNG_FIXED_EXPORT(233, void, png_set_cHRM_XYZ_fixed, (png_const_structrp png_ptr,
     png_fixed_point int_blue_Z))
 #endif
 
+#ifdef PNG_cICP_SUPPORTED
```

Comment:
These look like new functions added to the interface which will require re-running the name mangling.